src/views/main.hbs

```diff
@@ -1 +1,19 @@
 <h1>This is the main page</h1>
+
+<img src="./assets/fried.svg">
+
+
+<section class="main-menu__nav">
+  <a href="">Food</a>
+  <div class="food main-menu__nav-content">
+    <a href ="/dish-add">I have leftovers!</a>
+    <a href ="/dish-listings">I need food!</a>
+  </div>
+
+    <a href="">Community</a>
```

Comment:
maybe this should be a button not a link? try: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details